lib/room.rb

```diff
@@ -1,12 +1,22 @@
-class Rom
+class Room
+
+  attr_reader :category
+  attr_accessor :is_painted
+  alias_method :is_painted?, :is_painted
```

Comment:
great use of an alias_method!  Another way to accomplish this would be to define a method explicitly:

```ruby
def is_painted?
  @is_painted
end
```